lib/puppet-strings/markdown/templates/classes_and_defines.erb

```diff
@@ -82,7 +82,7 @@ Options:
 Default value: `<%= value_string(defaults[param[:name]]) %>`
 
 <% end -%>
-<% if hiera_overrides[param[:name]] -%>
+<% if !hiera_overrides.empty? && hiera_overrides[param[:name]] -%>
```

Comment:
I didn't have the empty check because in Ruby a missing key in a hash returns nil so it wouldn't be triggered. I think this change is redundant.

Reply:
`hiera_overrides` by default returns `[]` and defined as `@return [Array[Tuple[String, Hash[String, String], Any]]]` not sure if this is by design or not but i got a stack trace when it was empty to to string can't cast to int (or something like that)